tasks/TakeOutGarbage.tex

```diff
@@ -1,43 +1,49 @@
-\section{Take Out the Garbage [House Holder]}
```

Comment:
Why are we removing the Scenario?

Reply:
Yeah I was at first thinking this is a leftover of when we had this every task as GPSR idea in mind. Are these scenarios mentioned in the general rules? I think they should be if we have them in the title.

Reply:
The scenarios are mentioned in §3.6.1 and §3.6.2. Manipulation tasks can only be chosen within *House Holder/Keeper* scenario, while HRI tasks belong to *Party Host*. Test should specify to which scenario they belong.

Reply:
Yeah definitely. I'll add it back in my PRs